health-sdk/sdk/src/main/res/values/dimens.xml

```diff
@@ -1,15 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+    <dimen name="ghs_small_1">1dp</dimen>
     <dimen name="ghs_small">4dp</dimen>
     <dimen name="ghs_medium">8dp</dimen>
     <dimen name="ghs_medium_12">12dp</dimen>
     <dimen name="ghs_large">16dp</dimen>
     <dimen name="ghs_large_20">20dp</dimen>
     <dimen name="ghs_large_24">24dp</dimen>
+    <dimen name="ghs_large_25">25dp</dimen>
```

Comment:
[Request] It`s just 1dp from `ghs_large_24` so let's avoid introducing a dimension for 25dp 🤓 . We don't need to be that accurate. Also usually dimensions are divisible by 2 or 4.